lib/cocoapods/installer/analyzer/target_inspector.rb

```diff
@@ -218,7 +218,10 @@ def compute_recommends_frameworks(target_definition, native_targets)
         #
         def compute_swift_version_from_targets(targets)
           versions_to_targets = targets.inject({}) do |memo, target|
-            versions = target.resolved_build_setting('SWIFT_VERSION').values
+            resolve_against_xcconfig = target.build_configuration_list.build_configurations.all? do |bc|
```

Comment:
yes there is a case where the xcconfig is not set for all the configuration lists of the target but meh

Reply:
this is kinda gross, please add a comment ?